.github/workflows/linter.yml

```diff
@@ -0,0 +1,30 @@
+name: linter
+
+on: [push]
+
+env:
+  NODE_VERSION: 10.x
+
+jobs:
+  linter:
+    runs-on: ubuntu-latest
```

Comment:
name: Lint - ${{ matrix.os }} - Node v${{ matrix.node-version }}

Reply:
run-on is specified os version.

Reply:
Useful when you fast look on results, faced many times with this

Reply:
sorry I can't understand. you can use the suggestion feature.

Reply:
Just add name for job :smile:

Reply:
```
 name: Lint - ${{ matrix.os }} - Node v${{ matrix.node-version }}

runs-on: ${{ matrix.os }}
```